src/main/java/javaClasses/homework_5/Heghine_Khachatryan/interface_part/Taxi.java

```diff
@@ -0,0 +1,102 @@
+package javaClasses.homework_5.Heghine_Khachatryan.interface_part;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class Taxi implements Bookable, Stoppable {
+
+    private String transportName;
+    private Address address;
+
+    public Taxi() {
+
+    }
+
+    public Taxi(String transportName, Address address) {
+        this.transportName = transportName;
+        this.address = address;
+    }
+
+    @Override
+    public String bookTicket() {
+        String str = "N" + address.getHomeNumber() + ", " + address.getStreet() + " ," + address.getCity()
+                + " ," + address.getRegion();
+        String regex = "^[N.0-9a-zA-Z\\s,]+$";
+        // valid address will be N homeNumber, Street, City, Region
```

Comment:
If you want to put certain pattern for checking, you can use doc-comments above the method to provide the information. It is also nice to have live-example of what should be the passing string, as a separate comment (like // e. g. N 13 Shirakatsi Gyumri)

Reply:
done